orbstation/code/antagonists/traitor/items/ebow.dm

```diff
@@ -0,0 +1,47 @@
+#define MAX_EBOW_CHARGE 6
+
+/datum/uplink_item/stealthy_weapons/crossbow
+	desc = "A short bow mounted across a tiller in miniature. \
+	Small enough to fit into a pocket or slip into a bag unnoticed. \
+	It will synthesize and fire bolts tipped with a debilitating \
+	toxin that will damage and disorient targets, causing them to \
+	slur as if inebriated. It can produce an infinite number \
+	of bolts, but takes time to automatically recharge after each shot. \
+	Additionally, its internal battery must be manually rewound after six shots."
```

Comment:
instead of appending your change after this i think it would read better if they were both together, as they sound like these things are related instead of "this weapon has a 2 second  fire delay and needs to be reloaded after 6 shits"

so maybe like

It automatically recharges the next bold after the previous one is fired, but after six shots the mechanism must be rewound.

Reply:
> needs to be reloaded after 6 shits"

hehe six shits

Reply:
hmmm maybe im less sober than i thought